packages/victory-candlestick/src/victory-candlestick.tsx

```diff
@@ -28,7 +28,7 @@ export interface VictoryCandlestickStyleInterface {
   data?: VictoryStyleObject;
   high?: VictoryStyleObject;
   highLabels?: VictoryLabelStyleObject | VictoryLabelStyleObject[];
-  labels?: VictoryLabelStyleObject | VictoryLabelStyleObject[];
+  labels?: VictoryLabelStyleObject | VictoryLabelStyleObject[] | boolean;
```

Comment:
I don't think this change is necessary, as the label isn't expecting a boolean anywhere else in the code, but leaving it here until review in case there is a desire use case that I'm unfamiliar with.